app/javascript/src/Dashboard/components/ApiFilter.jsx

```diff
@@ -10,7 +10,14 @@ import { render } from 'react-dom'
 
 import 'Dashboard/styles/dashboard.scss'
 
-const ApiFilter = ({ apis, displayApis }) => {
+import type { Api } from 'Types'
+
+type Props = {
+  apis: Api[],
+  displayApis: Api[] => void
+}
+
+const ApiFilter = ({ apis, displayApis }: Props) => {
```

Comment:
You are returning a React div element, then you can:
```suggestion
import type { Element } from 'react'

const ApiFilter = ({ apis, displayApis }: Props): Element<'div'> => {
```